cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+
+    cy.visit('https://conduit.mate.academy/user/register');
+
+    cy.get(':nth-child(1) > .form-control').type(user.username);
+    cy.get(':nth-child(2) > .form-control').type(user.email);
+    cy.get(':nth-child(3) > .form-control').type(user.password);
 
+    cy.get('.btn').click();
```

Comment:
The '.btn' selector is too general and might lead to selecting the wrong button if another one is added in the future. Try to use a more specific selector.